vinculacion/vinculacion_main.py

```
@@ -0,0 +1,150 @@
+import pandas as pd
+from datetime import datetime, timedelta
```

Ruff (0.11.9)

`datetime.datetime` imported but unused

Remove unused import

(F401)